src/Symfony/Component/Routing/Loader/Configurator/ImportConfigurator.php

```diff
@@ -40,7 +41,7 @@ public function __destruct()
      *
      * @return $this
      */
-    final public function prefix($prefix, string $namePrefix = '')
+    final public function prefix($prefix, string $namePrefix = '', bool $trailingSlashOnRoot = true)
```

Comment:
bool arguments to change behavior are usually a bad design. an alternative would be to create a separate method like `removeTrailingSlash(string $path)` that does exactly what is currently implemented but in a separate method that reads like prose instead of some non-saying argument.